Code/Tools/AssetProcessor/assetprocessor_static_files.cmake

```diff
@@ -102,6 +102,8 @@ set(FILES
     native/utilities/PlatformConfiguration.cpp
     native/utilities/PlatformConfiguration.h
     native/utilities/PotentialDependencies.h
+    native/utilities/SimpleStatsCapture.cpp
+    native/utilities/SimpleStatsCapture.h
```

Comment:
Imho call these `StatsCapture` because they have a habit of not remaining simple if they're useful.